modules/ocf_mirrors/templates/systemd/sync.timer.erb

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Sync timer for <%= @title %>
+Description=<% if @type == "sync" %>Sync<% elsif @type == "monitor" %>Monitoring<% end %> timer for <%= @title %>
```

Comment:
I'd just set the `type` variable accordingly and do `Description=<% @type %>`. If not just split the conditionals over multiple lines for readability.

Reply:
Yeah I'd just do `Description=<%= @type %> timer for <%= title %>`, it makes enough sense and doesn't involved logic like this.